src/pages/History/History.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const History = () => {
+  return <div>History</div>;
+};
```

Comment:
Remove the history CSS empty file if it's not needed

Reply:
This is an old commit, the latest commits/changes have the history page functionalities and the history.css does contain styles for the same.